.gitignore

```diff
@@ -1,2 +1,3 @@
 node_modules
 dist/
+yarn.lock
```

Comment:
Yes, typically it is best to include it.  But in this *specific* repo I don't want to because it's not strictly required for electron-updater to work.  And someone who's new to Node/Electron development (like I am) gets confused when there are extra things.  I don't want anyone to think that `yarn.lock` is required.

Thanks for reviewing this, btw.